NEWS.md

```diff
@@ -33,6 +33,11 @@ from a BDS dataset, e.g. ADLB. (#943)
 
 ## Updates of Existing Functions
 
+- `impute_dtc` - Fixed imputation bug. A user setting `date_imputation = MID` and 
```

Comment:
Please use `impute_dtc()`. Then a link to the function is provided.

Reply:
You are thorough!  Thanks for catching this and making admiral so grand!!